qemu/tests/cfg/multi_host.cfg

```diff
@@ -78,12 +62,23 @@
                 # Migration properties
                 - @default:
                     type = migration_multi_host
+                    variants:
+                        - after_login_vm:
+                        - early_boot_vm:
```

Comment:
In my opinion,
we can remove @default
just: 
- after_login_vm: 
-early_boot_vm: